### ISSUE TITLE
Main navigation toggle button cannot receive focus, and collapsing menu loses focus

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/lively#17]
Lively is facing the same main navigation toggle button focusing issues as described in [omeka-s-themes/freedom#37](https://github.com/omeka-s-themes/freedom/issues/37).

**Comment by nelsonamaya82** [comment]
@aviannamiller this is fixed in the `master` branch, please double check and let me know if it works as expected.
